### PR TITLE
Update copyright info to "The SMP Authors"

### LINKED ISSUE #199: Update copyright information in the footer
**Current State of the service-mesh-performance site**
The footer of the website says "© 2021 Layer5, Inc." -

Screenshot_20210627_202928

**Desired State of the service-mesh-performance site**
Instead it should say "© 2021 The Service Mesh Performance Authors"

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat <navendupottekkat@gmail.com>

**Description**

This PR fixes #199 

This PR changes all the occurrences of "© 2021 Layer5, Inc." to "© 2021 The SMP Authors".

These include the footer in the website and the Licenses.

Screenshot_20210627_220942

- [x] Yes, I signed my commits.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2019, 2020 Layer5, Inc.
+   Copyright 2019, 2020, 2021 The SMP Authors
```
Comment (leecalcote) [inline LICENSE:189]:
```suggestion
   Copyright 2019, 2020, 2021 The Service Mesh Performance Authors
```

File: docs/LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright 2021, Layer5, Inc.
+Copyright 2021, The SMP Authors
```
Comment (leecalcote) [inline docs/LICENSE:1]:
```suggestion
Copyright 2021, The Service Mesh Performance Authors
```

File: docs/_includes/footer.html [hunk 3]
```diff
@@ -34,7 +34,7 @@ <h6 class="rspnsv">Service Mesh Performance</h6>
       <div class="container">
         <div class="row">
           <div class="col-sm-6 text-left">
-            <p class="l5-text">Copyright © Layer5, Inc. <script>document.write(new Date().getFullYear());</script></p>
+            <p class="l5-text">Copyright © The SMP Authors <script>document.write(new Date().getFullYear());</script></p>
```
Comment (leecalcote) [inline docs/_includes/footer.html:37]:
```suggestion
            <p class="l5-text">Copyright © The Service Mesh Performance Authors <script>document.write(new Date().getFullYear());</script></p>
```

File: docs/assets/js/back-to-top-smp.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Layer5 Authors
+// Copyright 2021 The SMP Authors
```
Comment (leecalcote) [inline docs/assets/js/back-to-top-smp.js:1]:
```suggestion
// Copyright 2021 The Service Mesh Performance Authors

```

**Comment by navendu-pottekkat** [comment]
@leecalcote committed your suggestions. I think we are good to go.

**Review by leecalcote — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
Addresses items in CNCF onboarding checklist: https://github.com/cncf/toc/issues/676

**Pull request merged**